plugins/source/aws/resources/services/rds/db_snapshots.go

```diff
@@ -55,40 +56,61 @@ func fetchRdsDbSnapshots(ctx context.Context, meta schema.ClientMeta, parent *sc
 		if err != nil {
```

Comment:
I might be missing something, but shouldn't the column resolver on line 41 be removed?

Reply:
If I do that, then the order of the columns changes, which I was trying to avoid

Reply:
Ah I see okay. We don't consider column ordering changes breaking but I guess it's better if we can not change it at all